deliver/lib/deliver/upload_screenshots.rb

```diff
@@ -9,17 +30,41 @@ def upload(options, screenshots)
       v = app.edit_version
       UI.user_error!("Could not find a version to edit for app '#{app.name}'") unless v
 
-      UI.message("Starting with the upload of screenshots...")
+      # md5's of uploaded screenshots
```

Comment:
this comment is slightly incorrect. It' the md5 of "screenshots to upload"

we could get rid of these comments anyway